### ISSUE TITLE
Puzzled about the calculation of energy MAE loss

### ISSUE THREAD

**Issue by whu-dft** [microsoft/Graphormer#142]
Thanks for your great work! Recently, I began to read the codes but encountered a problem. In the IS2RE task, I found the way to calculate the energy loss is in the following codes [link](https://github.com/microsoft/Graphormer/blob/main/graphormer/criterions/mae_deltapos.py#L52).

```python
        relaxed_energy = sample["targets"]["relaxed_energy"]
        relaxed_energy = relaxed_energy.float()
        relaxed_energy = (relaxed_energy - self.e_mean) / self.e_std
        sample_size = relaxed_energy.numel()
        loss = F.l1_loss(output.float().view(-1), relaxed_energy, reduction="none")
        ...
        loss = loss.sum()
```

Then when print the loss, you do the following conversion [link](https://github.com/microsoft/Graphormer/blob/main/graphormer/criterions/mae_deltapos.py#L93):

```python
        loss_sum = sum(log.get("loss", 0) for log in logging_outputs)
        ...
        sample_size = sum(log.get("sample_size", 0) for log in logging_outputs)
        mean_loss = (loss_sum / sample_size) * IS2RECriterion.e_std
```

After careful consideration，I can't agree that the above result is equal to the original meaning of MAE of energy, which should simply be :

```python
loss_sum = F.l1_loss(output, relaxed_energy, reduction="none").sum()
loss = loss_sum / sample_size
```

Since this problem should be important, hope to receive your kindly response. 

Best regards.

**Comment by zhengsx** [comment]
Would you elaborate more about your question?

**Comment by whu-dft** [comment]
@zhengsx Hi, I feel sorry to tell you that I have made a mistake. The way Graphormer calculates energy MAE loss is quit right. It is me who misunderstanded the method to handle it.
